flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/InputProcessorUtil.java

```diff
@@ -53,9 +54,11 @@ public static CheckpointBarrierHandler createCheckpointBarrierHandler(
 			}
 
 			if (taskManagerConfig.getBoolean(NetworkEnvironmentOptions.NETWORK_CREDIT_MODEL)) {
-				barrierHandler = new BarrierBuffer(inputGate, new CachedBufferBlocker(inputGate.getPageSize()), maxAlign);
+				barrierHandler = new BarrierBuffer(
+					inputGate, new CachedBufferBlocker(inputGate.getPageSize()), maxAlign, taskName);
 			} else {
-				barrierHandler = new BarrierBuffer(inputGate, new BufferSpiller(ioManager, inputGate.getPageSize()), maxAlign);
+				barrierHandler = new BarrierBuffer(
+					inputGate, new BufferSpiller(ioManager, inputGate.getPageSize()), maxAlign, taskName);
```

Comment:
codestyle: let's break all function argument into separate lines if the line is broken.